docs/topics/Gateway.md

```diff
@@ -819,15 +820,15 @@ You can use the `chunk_index` and `chunk_count` to calculate how many chunks are
-| members     | array of [guild member](#DOCS_RESOURCES_GUILD/guild-member-object) objects | set of guild members                                                                               |
-| chunk_index | integer                                                                    | the chunk index in the expected chunks for this response (0 <= chunk\_index < chunk\_count)        |
-| chunk_count | integer                                                                    | the total number of expected chunks for this response                                              |
-| not_found?  | array                                                                      | if passing an invalid id to `REQUEST_GUILD_MEMBERS`, it will be returned here                      |
-| presences?  | array of [presence](#DOCS_TOPICS_GATEWAY/presence) objects                 | if passing true to `REQUEST_GUILD_MEMBERS`, presences of the returned members will be here         |
-| nonce?      | string                                                                     | the nonce used in the [Guild Members Request](#DOCS_TOPICS_GATEWAY/request-guild-members)          |
+| Field       | Type                                                                       | Description                                                                                 |
+|-------------|----------------------------------------------------------------------------|---------------------------------------------------------------------------------------------|
+| guild_id    | snowflake                                                                  | the id of the guild                                                                         |
+| members     | array of [guild member](#DOCS_RESOURCES_GUILD/guild-member-object) objects | set of guild members                                                                        |
+| chunk_index | integer                                                                    | the chunk index in the expected chunks for this response (0 <= chunk\_index < chunk\_count) |
+| chunk_count | integer                                                                    | the total number of expected chunks for this response                                       |
+| not_found?  | array                                                                      | if passing an invalid id to `REQUEST_GUILD_MEMBERS`, it will be returned here               |
+| presences?  | array of [presence](#DOCS_TOPICS_GATEWAY/presence) objects                 | if passing true to `REQUEST_GUILD_MEMBERS`, presences of the returned members will be here  |
+| nonce?      | string                                                                     | the nonce used in the [Guild Members Request](#DOCS_TOPICS_GATEWAY/request-guild-members)   |
 
```

Comment:
Didn't you mention a while back that you only get the members through on this if you have presences intents enabled? Should this be documented here if this is intended behavior?

Reply:
uh, looks like you commented on the Guild Members Chunk area, which is the response to Request Guild Members, which only needs the presence intent to return presences. If you were talking about Guild Create, that's [already documented](https://discord.com/developers/docs/topics/gateway#guild-create) in the right place.

Reply:
oh, so i did. My mistake.